File: CS/CaptionStyleSelector.cs
using DevExpress.Xpf.Docking;
using System;
using System.Linq;
using System.Windows;
using System.Windows.Controls;

namespace WpfApplication1
{
    public class CaptionStyleSelector : StyleSelector
    {
        public Style AddNewTabStyle { get; set; }
        public override Style SelectStyle(object item, DependencyObject container) {
            if (item is ContentItem && ((ContentItem)item).Content is AddNewTabViewModel)
                return AddNewTabStyle;
            return base.SelectStyle(item, container);
        }
    }
}

File: VB/CaptionStyleSelector.vb
Imports DevExpress.Xpf.Docking
Imports System
Imports System.Linq
Imports System.Windows
Imports System.Windows.Controls

Namespace WpfApplication1
	Public Class CaptionStyleSelector
		Inherits StyleSelector

		Public Property AddNewTabStyle() As Style
		Public Overrides Function SelectStyle(ByVal item As Object, ByVal container As DependencyObject) As Style
			If TypeOf item Is ContentItem AndAlso TypeOf DirectCast(item, ContentItem).Content Is AddNewTabViewModel Then
				Return AddNewTabStyle
			End If
			Return MyBase.SelectStyle(item, container)
		End Function
	End Class
End Namespace
